schemas/interface.ts

```diff
@@ -1,18 +1,186 @@
-
-/**
- * Question objects used to create a quiz. A Quiz is a set of questions
- * 
- * @version 0.1.0
- */
-export interface Question {
+declare namespace Quizoot {
     /**
-     * Id of the question
+     * Question object used to create a quiz. A Quiz is a set of questions.
      * 
-     * @pattern [a-z0-9]{12}
+     * @version 0.1.0
+     */
+    export interface Question {
+        /**
+         * Id of the question
+         * 
+         * @pattern [a-z0-9]{12}
+         */
+        id: string;
```

Comment:
Why do you use strings instead of numbers for your `id` props ?

Reply:
Good point. We have more ids if we use strings haha. I don't know how many questions we might have so.. just to be extremely safe

Reply:
A database of billions of questions 🤣 why not